tests/test_maintel_track_target.py

```diff
@@ -151,6 +164,15 @@ async def test_configure_with_ra_dec(self):
             # Script can be configured with ra, dec only
             await self.configure_script(slew_icrs=dict(ra=1.0, dec=-10.0))
 
+    async def test_configure_with_ra_dec_sexagesimal(self):
+        async with self.make_script():
+            self.script._mtcs = unittest.mock.AsyncMock()
+            self.script._mtcs.start_task = utils.make_done_future()
+            # Script can be configured with ra, dec only
+            await self.configure_script(
+                slew_icrs=dict(ra="+0:00:00", dec="-10:00:00.0")
```

Comment:
This doesn't quite enforce sexagesimal since it should be "+00:00:00" for ra value, since we need two values in the hr column?

Reply:
Not sure I understood your comment `"+0:00:00"` is a valid entry.

Reply:
I mean for strict hexagesimal compliance it would need to be "+00:00:00"...but as long as it complies with slew_icrs I think we are fine.

Reply:
Actually, I think I'm wrong about needing the second leading 0 for hexagesimal compliance...feel free to ignore this request.